src/actions/registerUserActions.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { REGISTER_USER_SUCCESS, REGISTER_USER_ERROR } from './types';
+
+export const fetchUsers = (postData) => dispatch => {
```

Comment:
Nice work on implementing this feature, @MuhanguziDavid.
This block of code is unclear to me. Is it necessary to have two arrow functions like this?